src/main/java/katheryne/Message.java

```diff
@@ -16,4 +18,6 @@ public class Message {
     public static final String MESSAGE_DATE_FORMAT_ERROR = "Katheryne: Please enter date in 'yyyy-MM-dd' format";
     public static final String MESSAGE_INDEX_ERROR = "Katheryne: That task number doesn't exist. Please try again.";
     public static final String MESSAGE_LOADING_ERROR = "Katheryne: No saved task found. Will start a new one.";
+    public static final String MESSAGE_FIND_RESULT = "Katheryne: Here are the matching tasks in your list:";
+    public static final String MESSAGE_TASK_NOT_FOUND = "Katheryne: There is no matching task in your list.";
```

Comment:
I like that your use of constants for common error messages!

Reply:
should these be an enum instead?